### PR TITLE
Expose metrics

### LINKED ISSUE #19: Expose metrics instead of sending to Pushgateway
https://github.com/thoth-station/advise-reporter/blob/f91b82d54d420e8f3790e18a7a830b6fe62b8c07/consumer.py#L40-L45

---

###### This issue was generated by [todo](https://todo.jasonet.co) based on a `TODO` comment in f91b82d54d420e8f3790e18a7a830b6fe62b8c07 when #15 was merged. cc @pacospace.

### PR DESCRIPTION
**Pull request by pacospace**
## Related Issues and Dependencies

Fixes: https://github.com/thoth-station/advise-reporter/issues/19

## This introduces a breaking change

- [ ] Yes
- [x] No

## This Pull Request implements

Remove pushgateway, introduce new metrics and expose them

### CODE REVIEW

File: thoth/advise_reporter/advise_reporter.py [hunk 1]
```diff
@@ -63,24 +50,13 @@ def retrieve_adviser_reports_justifications(adviser_version: str):
...
-
-
+@metrics.exceptions.count_exceptions()
+@metrics.in_progress.track_inprogress()
 def expose_metrics(advise_justification: MessageBase):
     """Retrieve adviser reports justifications."""
-    _METRIC_ADVISE_TYPE.labels(advise_message=advise_justification.message, thoth_environment=_THOTH_ENVIRONMENT).set(
-        advise_justification.count
-    )
+    metrics.advise_justification_type.labels(
```
Comment (KPostOffice) [inline thoth/advise_reporter/advise_reporter.py]: I'm confused about what this is exposing

**Review by KPostOffice — COMMENTED** [review]
Just one small comment

File: thoth/advise_reporter/advise_reporter.py (hunk 1 above)
Comment (pacospace) [reply thoth/advise_reporter/advise_reporter.py]: It's exposing the number of justifications provided by adviser per type of justification, so we can monitor what we provide and help in deciding what needs to be prioritized

**Review by KPostOffice — APPROVED** [review]

**Pull request merged**